### PR TITLE
[CWS] Fix CSM linter

### PR DESCRIPTION
**Pull request by spikat**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/secl/compiler/eval/opts.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ type VariableStore struct {
 	Variables map[string]VariableValue
 }
 
-// AddVariable add a variable
+// Add add a variable
```
Comment (modernplumbing) [inline pkg/security/secl/compiler/eval/opts.go:48]: Add adds a variable

File: pkg/security/secl/model/model_unix.go [hunk 2]
```diff
@@ -549,6 +549,7 @@ type SELinuxEvent struct {
 }
 
 const (
+	// ProcessCacheEntryFromUnknown and subsequent defines the possible source for process cache entries
```
Comment (modernplumbing) [inline pkg/security/secl/model/model_unix.go:552]: define, not defines

File: pkg/security/secl/rules/collected_events_regular.go [hunk 3]
```diff
@@ -9,9 +9,11 @@ package rules
 
 import "github.com/DataDog/datadog-agent/pkg/security/secl/compiler/eval"
 
+// EventCollector defines an event collector
 type EventCollector struct {
 }
 
+// CollectEvent collect event
```
Comment (modernplumbing) [inline pkg/security/secl/rules/collected_events_regular.go:16]: collects (also feel free to ignore these nits)

**Review by modernplumbing — APPROVED** [review]
Left a couple grammatical suggestions that can be safely ignored

File: pkg/security/secl/model/model_unix.go (hunk 2 above)
Comment (spikat) [reply pkg/security/secl/model/model_unix.go:552]: thanks

**Comment by spikat** [comment]
>

**Comment by spikat** [comment]
@modernplumbing thanks for your suggestions, applied in last commit :)

**Review by YoannGh — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Comment by brycekahle** [comment]
@spikat I think you only need package docs in a single file per package. From https://tip.golang.org/doc/comment

> For multi-file packages, the package comment should only be in one source file. If multiple files have package comments, they are concatenated to form one large comment for the entire package.

**Comment by spikat** [comment]
> @spikat I think you only need package docs in a single file per package. From https://tip.golang.org/doc/comment
> 
> > For multi-file packages, the package comment should only be in one source file. If multiple files have package comments, they are concatenated to form one large comment for the entire package.

Arf .. I've just checked and it seems that's right, my sed was a bit wild I guess ^^

**Review by nplanel — APPROVED** [review]

File: pkg/util/trivy/cache.go [hunk 4]
```diff
@@ -100,28 +101,28 @@ type Cache interface {
 	Len() int
 }
 
-// TrivyCache holds a generic Cache and implements cache.Cache from Trivy.
-type TrivyCache struct {
+// TCache holds a generic Cache and implements cache.Cache from Trivy.
```
Comment (AliDatadog) [inline pkg/util/trivy/cache.go:104]: I think `TCache` here isn't meaningful. Can we change that ?

Comment (spikat) [reply pkg/util/trivy/cache.go:104]: I agree.. The issue here was that the linter didn't want to have a trivy.TrivyCache, asking rename it to trivy.Cache .. but there is already a such type name .. So, I land with this poor one but, happy to hear if you have suggestions :)

Comment (AliDatadog) [reply pkg/util/trivy/cache.go:104]: Can we rename it as ScannerCache ?

Comment (spikat) [reply pkg/util/trivy/cache.go:104]: Sure, let me change that

**Review by AliDatadog — APPROVED** [review]

**Pull request merged**